tools/mountsnoop.py

```diff
@@ -157,6 +162,9 @@
     bpf_get_current_comm(event.enter.comm, sizeof(event.enter.comm));
     event.enter.flags = flags;
     task = (struct task_struct *)bpf_get_current_task();
+    event.enter.ppid = task->real_parent->tgid;
+    __builtin_memcpy(event.enter.pcomm, task->real_parent->comm, TASK_COMM_LEN);
```

Comment:
can you use `bpf_probe_read_kernel` here, and in the other `__builtin_memcpy` instance?

Reply:
OK. Thanks for your advice.